nss-day-cohort-60/national-parks-client
# Login modal on nav bar

# Description
1. Added a react bootstrap modal to login on nav bar 
2. Added logo placeholder on nav bar and some minor styling
3. Added a welcome message to the user and a "My Blogs" link when the user logs in
4. Added list of natural attractions to ParksPage.js
5. Added NavBar to index.js so that it appears on every page
6. Added CDNs to index.html for bootstrap

# Steps to test

1. Fetch the md-user-management branch and switch to it
2. Start/restart debugger
3. Make sure react bootstrap and bootstrap are installed on your computer
4. Start the client in the browser
5. Click on login on the nav bar and verify that a modal is displayed.
6. Login with an existing user email (frodo@baggins.com is hardcoded there right now)
7. Verify that the user is logged in. Should see a welcome message on the nav bar and "My Blogs" should also be displayed when logged in.
8. Click the logout link and verify that the user is logged out (logout link, My Blogs, and welcome message should no longer be displayed).
9. Navigate to "home" and click on a park link. Verify that the natural attractions for that park are displayed at the bottom of the page.
